README.md

```diff
@@ -62,6 +64,15 @@ these lines to your `.zshrc`:
 If you are enabling `url-quote-magic`, make sure to load it first and
 then load `git-escape-magic`.
 
+
+### Zinit
+
+If you are using [Zinit](https://github.com/zdharma-continuum/zinit),
+you can load git-escape-magic by adding the following line to your `.zshrc`:
+```zsh
+zinit snippet https://github.com/knu/zsh-git-escape-magic/blob/master/git-escape-magic
```

Comment:
If I had to advertise a fixed URL, I'd prefer the branch name to be called `main`.  And I've just changed it [so](https://github.com/knu/zsh-git-escape-magic/tree/main).

```suggestion
zinit snippet https://github.com/knu/zsh-git-escape-magic/blob/main/git-escape-magic
```

I'll keep the `master` branch for awhile, but it will eventually go.